src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  public color: Color;
+
+  private pointA: number;
+
+  private pointB: number;
+
+  private pointC: number;
+
+  constructor(color: Color, pointA: number, pointB: number, pointC: number) {
```

Comment:
```suggestion
  public color: Color;

  constructor(private color: Color, private pointA: number,private  pointB: number, private pointC: number) {
```
You can add modifiers directly in constructor and in this case you don't have to assign values, typescript will do it for you